tutorials/navigation/navigation_introduction_2d.rst

```diff
@@ -13,6 +13,12 @@ Godot provides the following objects and classes for 2D navigation:
 
     The AStar2D class is best suited for cell-based 2D gameplay that does not require actors to reach any possible position within an area but only predefined, distinct positions.
 
+- :ref:`AstarGrid2D<class_AstarGrid2D>`
+    ``AstarGrid2D`` objects provide an option to find the shortest path between two points on a partial 2D grid.
+
+    The AStarGrid2D class is best suited for cell-based 2D gameplay that only require actors to move between distinct positions on a grid. 
+    AStarGrid2D is better used over AStar2D when applicable as it only requires the region of the grid to be set, rather than manually creating and connecting individual points.
```

Comment:
Hi, hello and welcome 👋 Well spotted, adding this is useful!

I found the long description in the class reference easier to parse and understand the pros/cons:

> AStarGrid2D is a variant of [AStar2D](https://docs.godotengine.org/en/stable/classes/class_astar2d.html#class-astar2d) that is specialized for partial 2D grids. It is simpler to use because it doesn't require you to manually create points and connect them together. This class also supports multiple types of heuristics, modes for diagonal movement, and a jumping mode to speed up calculation

Reply:
I made some changes based on your feedback, apologies for the long wait.